ruby/vampires.rb

```diff
@@ -0,0 +1,68 @@
+def foodMethod (food)
+  if food=="yes"
+    return true
+  elsif food=="no"
+    return false
+  end
+end
+
+def insuranceMethod(insurance)
+  if insurance=="yes"
+    return true
+  elsif insurance=="no"
+    return false
+  end
+end
+
+puts "How many employees would you like to process?"
+amount=gets.chomp.to_i
+result=""
+
+while (amount > 0)
+  puts "What is your name?"
+  name=gets.chomp 
+  puts "How old are you?"
+  age=gets.chomp.to_i
+  puts "What year were you born?"
+  year=gets.chomp.to_i
+  puts "Our company cafeteria serves garlic bread. Should we order some for you? (yes/no)"
+  food=gets.chomp
+  food=foodMethod(food)
+  puts "Would you like to enroll in the company's health insurance? (yes/no)"
+  insurance=gets.chomp
+  insurance=insuranceMethod(insurance)
+  answer=""
+  puts "Do you have any allergies? (type 1 at a time until finished, type 'done' when finished or if no allergies)"
+  while answer!="done"
+    answer=gets.chomp
+    if answer=="sunshine"
+      result="Probably a vampire"
+      break
+    end
+  end
+
+  yearDiff=2016-age

```

Comment:
with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`